spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe 'PostIndex', type: :feature do
+  let(:user) { FactoryBot.create(:user) }
+  let(:post) { FactoryBot.create(:post, author: user) }
+  let(:comments) { FactoryBot.create_list(:comment, 3, post:, text: 'old', author: user, created_at: 4.days.ago) }
+
+  let(:recent_comments) { FactoryBot.create_list(:comment, 5, post:, text: 'new', author: user) }
+
+  before do
+    post
+    comments
+    recent_comments
+    visit user_posts_path(user)
+  end
+
+  describe 'User section in user posts index' do
+    it 'shows user name' do
+      expect(page).to have_content(user.name)
+    end
+
+    it 'shows user photo' do
+      expect(page).to have_css("img[alt='User Photo']")
+    end
+
+    it 'shows user posts count' do
+      expect(page).to have_content(user.posts_counter)
+    end
+  end
+
+  describe 'Posts section in user posts index' do
+    it 'Shows post heading' do
+      expect(page).to have_content('Post📝')
+    end
+
+    it 'Shows post content' do
+      expect(page).to have_content(post.text)
+    end
+
+    it 'Shows pagination section if there are more posts than fit on the view' do
+      FactoryBot.create_list(:post, 10, author: user)
+      visit user_posts_path(user)
+      expect(page).to have_content('Pagination')
+    end
+
```

Comment:
- [ ] While you have done a good job, this file should contain test for the number of likes a post has, kindly add it as it is a requirement.
![image](https://github.com/sadaf-Daneshgar/blogapp/assets/86200356/4a4968b3-77e9-491b-9d67-828e11fd0be0)